config/configtelemetry/doc.go

```diff
@@ -4,4 +4,51 @@
 // Package configtelemetry defines various telemetry level for configuration.
 // It enables every component to have access to telemetry level
 // to enable metrics only when necessary.
+//
+// This document provides guidance on which telemetry level to adopt for Collector metrics.
+// When adopting a telemetry level, the developer is expected to rely on this guidance to
+// justify their choice of telemetry level.
+//
+// 1. configtelemetry.None
+//
+// No telemetry data should be collected.
+//
+// 2. configtelemetry.Basic
+//
+// Signals associated with this level cover the essential coverage of the collector telemetry.
+// It should only be used for internal collector telemetry.
+//
+// This is the default level recommended when running the Collector.
```

Comment:
Currently, Normal is the default. Are we going to change that? I'd suggest not changing it but keeping this level lean instead

Reply:
fixed